anomaly_detector/fact_store/fact_store_api.py

```diff
@@ -33,6 +34,7 @@ def write_event(self, predict_id, message, score, anomaly_status):
         event = EventModel(message=message, score=score, predict_id=predict_id, anomaly_status=anomaly_status)
         self.session.add(event)
         self.session.commit()
+        return True
         print("Event ID: {}  recorded in events_store".format(event.predict_id))
```

Comment:
The code will never get to the `print`, you can remove it or move above return ...also probably should be `_LOGGER.debug`

Reply:
Good catch I'll move this down

Reply:
FIxed.